test/WebSites/FiltersWebSite/Controllers/ResourceFilterController.cs

```diff
@@ -31,7 +33,9 @@ private class ShortCircuitWithFormatterAttribute : Attribute, IResourceFilter
 
             public ShortCircuitWithFormatterAttribute()
             {
-                _formatters = new IOutputFormatter[] { new JsonOutputFormatter() };
+                _formatters = new IOutputFormatter[] { new JsonOutputFormatter(
+                    new JsonSerializerSettings(),
```

Comment:
(I suck at these emojis :-)...I did a +1 unintentionally...how do i revert that back?...doing  a -1 created one more entry :-))
The instance of serializer settings on `MvcJsonOptions` is shared across all controllers, so it cannot be used on `JsonFormatterController` as it is mutating it. Probably have a comment here about `MvcJsonOptions` here.